test/api/ticket/api_test.go

```diff
@@ -0,0 +1,164 @@
+package ticket
+
+import (
+	"testing"
+
+	"github.com/konveyor/tackle2-hub/api"
+	TrackerSamples "github.com/konveyor/tackle2-hub/test/api/tracker"
+	"github.com/konveyor/tackle2-hub/test/assert"
+)
+
+func TestTicketCRUD(t *testing.T) {
+	for _, r := range Samples {
+		t.Run("Ticket "+r.Kind+" CRUD", func(t *testing.T) {
+
+			// Create a sample Application for the ticket.
+			app := api.Application{
+				Name: r.Application.Name,
+			}
+			assert.Must(t, Application.Create(&app))
+
+			createdIdentities := []api.Identity{}
+			createdTrackers := []api.Tracker{}
+			for _, tracker := range TrackerSamples.Samples {
+				// Create a sample identity for the tracker
+				identity := api.Identity{
+					Name: tracker.Identity.Name,
+					Kind: tracker.Kind,
+				}
+				assert.Must(t, Identity.Create(&identity))
+				createdIdentities = append(createdIdentities, identity)
+				assert.Must(t, Tracker.Create(&tracker))
+				createdTrackers = append(createdTrackers, tracker)
+			}
+
+			// Create a sample ticket
+			assert.Must(t, Ticket.Create(&r))
```

Comment:
The same with quite unreliable link from Ticket to Tracker (ID=1) when one line assignment could set it. But not merge blocker in this case.